src/Applications/rank_object_enumerator.py

```diff
@@ -197,14 +270,73 @@ def compute_arrangement_works(objects: tuple, arrngmnt: tuple, alpha_c: float, b
         # Keep track of number of arrangements for sanity
         n_arrangements += 1
 
-    # Sanity check
-    LGR.info(f"Number of generated arrangements: {n_arrangements}")
-    if n_arrangements != N_RANKS ** len(objcts):
-        LGR.error("Incorrect number of arrangements")
+    # Return quantities of interest
+    return n_arrangements, works_min_max, arrangements_min_max
+
+
+def recursively_compute_transitions(visited, objects, arrangement, alpha: float, beta: float, gamma: float, w_max: float, w_min_max: float, n_ranks:int, max_objects:int=None):
```

Comment:
Same remark as for the docstrings. For now we have the convention that function definitions are on a single line. We should replace **all of them** at once. This would deserve a separate issue.